suite-native/module-onboarding/src/screens/FeatureReceiveScreen.tsx

```diff
@@ -78,7 +80,7 @@ const FeatureReceiveScreenNoDevice = () => {
 };
 
 export const FeatureReceiveScreen = () => {
-    const { isUsbDeviceConnectFeatureEnabled } = useIsUsbDeviceConnectFeatureEnabled();
+    const isUsbDeviceConnectFeatureEnabled = useSelector(selectIsDeviceConnectFeatureFlagEnabled);
```

Comment:
Why did you deleted our custom hook? One of the key points why we had that hooks to avoid these kind of refactoring everywhere instead you should just use this selector once in the hook and keep it. Please bring it back.

Reply:
Yes, I agree with that and left it like that but then I thought maybe I'm just being lazy and could refactor it as there is no need for this hook anymore and likely won't be as well. But never mind, I will put it back.

Reply:
https://github.com/trezor/trezor-suite/pull/9840/commits/5f26d8a42f7657d579b87b15fda7576d67d39060